### PR TITLE
EFRS-1102: Clone recognition service with uploaded images

### PR DESCRIPTION
**Pull request by bestDvlpr**

### CODE REVIEW

**Comment by ksidikov** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by ksidikov** [comment]
recheckcla

File: java/admin/src/main/resources/application.yml [hunk 1]
```diff
@@ -46,6 +46,7 @@ spring:
     database: postgresql
     open-in-view: true
     generate-ddl: false
+    show-sql: true
```
Comment (pospielov) [inline java/admin/src/main/resources/application.yml:49]: Can you remove it for the final version?

Comment (bestDvlpr) [reply java/admin/src/main/resources/application.yml:49]: done!

File: java/admin/src/main/java/com/exadel/frs/service/ModelService.java [hunk 2]
```diff
@@ -166,56 +169,44 @@ public Model cloneModel(
         return cloneAppModelAccessList;
     }
 
-    @Transactional
     public void cloneSubjects(final String sourceApiKey, final String newApiKey) {
```
Comment (pospielov) [inline java/admin/src/main/java/com/exadel/frs/service/ModelService.java:172]: Will it work in one transaction? So if one of the subject's cloning fails, will it remove the already cloned subjects?

**Review by pospielov — APPROVED** [review]

**Pull request merged**